manifests/apache.pp

```diff
@@ -2,9 +2,9 @@
 # @api private
 class pulpcore::apache {
   $api_path = '/pulp/api/v3'
-  $api_url = "http://${pulpcore::api_host}:${pulpcore::api_port}${api_path}"
+  $api_url = "${pulpcore::api_service_bind}|http://${pulpcore::servername}${api_path}"
   $content_path = '/pulp/content'
-  $content_url = "http://${pulpcore::content_host}:${pulpcore::content_port}${content_path}"
+  $content_url = "${pulpcore::content_service_bind}|http://${pulpcore::servername}${content_path}"
```

Comment:
This will break things in puppet-foreman_proxy_content:
https://github.com/theforeman/puppet-foreman_proxy_content/blob/743517dac849b5a0cd9bec56077f50f227a1f8c5/templates/pulpcore-docker-apache.conf.erb#L6-L10

In https://github.com/theforeman/puppet-pulpcore/pull/105 I hope to address that, but I haven't been able to sit down and even check out the branch to update it in the past weeks.

Reply:
Ahh yes indeed, you solved a testing mystery for me (there was one test that failed and that was pulling container content)